apps/condo/domains/help/access/Help.js

```diff
@@ -0,0 +1,55 @@
+// @ts-nocheck
```

Comment:
It's not a new domain. Move it inside the Ticket. It's a hint when creating/updating a ticket.
Probably you can use `TicketHint` name or something similar